src/main/java/pt/tqs/g205/services/PratoService.java

```diff
@@ -0,0 +1,30 @@
+package pt.tqs.g205.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import pt.tqs.g205.domain.Prato;
+import pt.tqs.g205.repositories.PratoRepository;
+
+import java.util.List;
+
+/**
+ * Servicos para a classe Prato.
+ */
+@Service
+public class PratoService {
+
+  @Autowired
+  private PratoRepository pratoRepo;
+  
+  /**
+   * Obter todos os pratos.
+   * @return lista de Pratos.
+   */
+  public List<Prato> getAll() {
+    List<Prato> pratos = pratoRepo.findAll();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Immediately return this expression instead of assigning it to the temporary variable "pratos". ([squid:S1488](https://sonarcloud.io/organizations/bdomingues-github/rules#rule_key=squid%3AS1488))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=pt.tqs.g205%3Arestapi&pullRequest=5&issues=AWNCwXmVXOBV_Yrh-xNx&open=AWNCwXmVXOBV_Yrh-xNx)

[ISSUE_KEY]: AWNCwXmVXOBV_Yrh-xNx